src/store/plugins/pollAccountInfo.js

```diff
@@ -11,8 +18,8 @@ export default async (store) => {
 
   function startPolling() {
     pollDefaultNames = setInterval(() => store.dispatch('names/setDefaults'), 10000);
-    pollAvailableTokens = setInterval(() => store.dispatch('fungibleTokens/loadAvailableTokens'), 60000);
-    pollTokenBalances = setInterval(() => store.dispatch('fungibleTokens/loadTokenBalances'), 10000);
+    pollAvailableTokens = setInterval(() => loadAvailableTokens(), 60000);
+    pollTokenBalances = setInterval(() => loadTokenBalances(), 10000);
```

Comment:
I think instead of constant polling we should use composable helper: `createPollingBasedOnMountedComponents` within the `fungibleTokens` composable so the polling will only happen when any of the App components would actually need this.